src/uiprotect/api.py

```diff
@@ -157,6 +158,42 @@ def get_user_hash(host: str, username: str) -> str:
     return session.hexdigest()
 
 
+class RTSPSStreams(ProtectBaseObject):
+    """RTSPS stream URLs for a camera."""
+
+    model_config = {"extra": "allow"}
```

Comment:
```suggestion
    model_config = {"extra": "allow"}
    high: str | None = None
    medium: str | None = None
    low: str | None = None
    ultra: str | None = None    
```

I think this would work here

Reply:
added code comment